src/quo2/screens/info/page_nav.cljs

```diff
@@ -0,0 +1,114 @@
+(ns quo2.screens.info.page-nav
```

Comment:
the namespace here is wrong, it should be in `navigation`.

Reply:
Done